doc/manpages/dinit.8.m4

```diff
@@ -298,6 +302,16 @@ There are several ways to work around this.
 Service names following the \fB\-\-container\fR (\fB\-o\fR) or \fB\-\-system\-mgr\fR (\fB\-m\fR) options are not ignored.
 Also, the \fB\-\-service\fR (\fB\-t\fR) option can be used to force a service name to be recognised regardless of operating mode.
 .\"
+.SH SELINUX SUPPORT
+.LP
+When running as PID 1 on a SELinux enabled machine, \fBdinit\fR will by default load the system's SELinux policy.
+This behaviour can be disabled by passing \fB\-\-disable\-selinux\-policy\fR to dinit through the kernel cmdline.
```

Comment:
This implies that putting the  option on the kernel command line is the way to disable the policy loading. But, putting it on the kernel command line might do nothing (it depends on initrd setup). Also, it doesn't have to be on the kernel command line to be effective (what if the initrd's init passes it unconditionally or reads options from a file or...?)

I.e. It doesn't matter how `--disable-selinux-policy` is passed to dinit. If it is passed to dinit, the behaviour will be disabled. So say that, don't specify unnecessary (and potentially incorrect) details.